packages/csv-parser-discount-code/src/main.js

```diff
@@ -40,14 +40,19 @@ export default class CsvParserDiscountCode {
     }
 
     this._cartDiscountsToArray = this._cartDiscountsToArray.bind(this)
+    this._groupsToArray = this._groupsToArray.bind(this)
     this._handleErrors = this._handleErrors.bind(this)
 
     this._rowIndex = 0
   }
 
   // Remove fields with empty values from the code objects
   static _removeEmptyFields(item: Object) {
-    return _.omitBy(item, val => val === '')
+    const fileredObj = {}
+    Object.keys(item).forEach(key => {
+      if (item[key] !== '') fileredObj[key] = item[key]
```

Comment:
use `reduce` 

*cough* transducers is fully possible here *cough*

Reply:
Using `acc` here is the only way to keep lint happy with regards to [this comment](https://github.com/commercetools/nodejs/pull/468#discussion_r169278076)